test/NNKolmogorov_tests.jl

```diff
@@ -1,35 +1,25 @@
 using Test, Flux, NeuralNetDiffEq , StochasticDiffEq
 using DiffEqDevTools
-#Initial case for u(0 , x)
-function phi(xi)
-    y = Float64[]
-    for x in xi
-        y = push!(y , 1.77*x -0.015*x^3)
-    end
-    y = reshape(y , size(xi)[1] , size(xi)[2] )
-    return y
-end
+using Distributions
 
-xspan = (-6.0 , 6.0)
+u0 = 0
+xspan = (-1.0 , 1.0)
 tspan = (0.0 , 1.0)
-#f = mu and g  = sigma
-g(u , p , t) = 0.5*u
-f(u , p , t) = 0.5*0.25*u
+g(u , p , t) = 1
+f(u , p , t) = 0
 d = 1
 sdealg = EM()
-prob = KolmogorovPDEProblem(f , g, phi , xspan , tspan, d)
+prob = SDEProblem(f , g , u0 , (0.0 , 1.0) ; xspan = xspan , d = d)
```

Comment:
Why was the previous test case removed? It would probably be good to have both?

Reply:
I added it again. And kept both the problems now as of the previous discussion